src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -872,64 +909,6 @@ private function buildTree()
                 ],
             ],
         ];
```

Comment:
It's probably a bad idea to do it like this. If the xml files are updated it has also to be done here, leads to hard to find errors for sure.